Description: Fix MIPS64 asm constraints
  Assembly problem, using =l instead of =lc.
  http://stackoverflow.com/questions/3652153/impossible-constraint-in-asm
Author: YunQiang Su <wzssyqa@gmail.com>
Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=798126
Forwarded: yes, http://bugs.mysql.com/69978
Last-Update: 2015-09-21

Index: mysql-5.6/extra/yassl/taocrypt/src/integer.cpp
===================================================================
--- mysql-5.6.orig/extra/yassl/taocrypt/src/integer.cpp
+++ mysql-5.6/extra/yassl/taocrypt/src/integer.cpp
@@ -192,7 +192,7 @@
                 "a" (a), "rm" (b) : "cc");
 
         #elif defined(__mips64)
-            __asm__("dmultu %2,%3" : "=h" (r.halfs_.high), "=l" (r.halfs_.low)
+            __asm__("dmultu %2,%3" : "=d" (r.halfs_.high), "=lc" (r.halfs_.low)
                 : "r" (a), "r" (b));
 
         #elif defined(_M_IX86)
